assignment09/part02/index.php

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 2</title>
+</head>
+<body>
+    
+    <?php
+        //Task 3.1: Change these items to reflect
+        //four of your favorite things
+        $my_favorite_things = array (
+            "My first favorite thing is my family.",
+            "My second favorite thing is my facebook group.",
+            "My third favorite thing is my friends.",
+            "My fourth favorite thing is my laptop."
+        );
+        $counter = 0;
+    ?>
+
+    <h1>My Favorite Things</h1>
+        <!-- Task 3.2: Code to output things below -->
+        <?php
+            do {
+                echo $my_favorite_things[$counter]. "<br />";
```

Comment:
nice loop! And you are outputting each element - but we're missing the ordered list. I'd expect it to start and end (<ol> & </ol>) before and after your loop and then create an <li> for each element here.